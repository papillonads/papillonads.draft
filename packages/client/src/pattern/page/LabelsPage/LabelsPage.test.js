import React from 'react'
import labelsPageDefault from './LabelsPage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<LabelsPage />', () => {
  describe('Snaphot', () => {
    test('must match labelsPageDefault', () => {
      expect(global.renderToJSON(labelsPageDefault)).toMatchSnapshot()
    })
  })
})
