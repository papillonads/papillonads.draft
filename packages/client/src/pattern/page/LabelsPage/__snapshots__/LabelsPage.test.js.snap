// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LabelsPage /> Snaphot must match labelsPageDefault 1`] = `<div />`;
